background/constants/networks.ts

```diff
@@ -612,6 +612,8 @@ return shardData[0].shard
 export function setProviderForShard(providers: SerialFallbackProvider): SerialFallbackProvider {
   let shard = globalThis.main.SelectedShard
   if (shard === undefined || shard == "") {
+    console.error("Shard is undefined")
```

Comment:
Likely unneeded console.log. Seems like the standard is to use the existing logger?

Reply:
The logger does not print a full stack trace, so I prefer to use console.log and console.error

Reply:
![Screen Shot 2023-08-01 at 10 33 40 AM](https://github.com/PelagusWallet/pelagus-extension/assets/26939949/27469c90-dc4f-464f-8dfd-e3f25c3eacad)